native/CMakeLists.txt

```diff
@@ -117,3 +117,9 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE "-DAWS_DOTNET_EXPORTS")
 target_include_directories(${PROJECT_NAME} PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
         $<INSTALL_INTERFACE:include>)
+
+# Strip debug symbols from s2n and libcrypto to keep the .so size down
+if (UNIX AND NOT APPLE AND CMAKE_BUILD_TYPE STREQUAL Release AND CMAKE_STRIP)
```

Comment:
wouldn't this only apply with RelWithDebInfo? Does release even have debug symbols?

Reply:
It does! I tested both configurations. I could not find a file size difference between RelWithDebInfo and Release, and I confirmed that they are being used when I forced the `CMAKE_BUILD_TYPE`